websites/C/Coursera/presence.ts

```diff
@@ -0,0 +1,350 @@
+						})(),
+					},
+				],
+			},
+			{
+				path: /^\/learn\/.*\/course-inbox$/,
+				smallImageKey: () => Assets.Reading,
+				smallImageText: () => "Learning",
+				state: () => {
+					return "Checking Course Inbox";
+				},
+				details: () => {
+					return document.location.href.split("/")[4].split("-").join(" ");
+				},
+				buttons: () => [
+					{
+						label: "View Course",
+						url: (() => {
+							return document.location.href.split("/").slice(0, 5).join("/");
+						})(),
+					},
+				],
+			},
+
+			{
+				path: /^\/learn\/.+\/lecture\/*/,
+				smallImageKey: () => Assets.Reading,
+				smallImageText: () => "Learning",
+				state: () => {
+					return `Lecture: ${document.title.substring(
+						0,
+						document.title.lastIndexOf(" ") - 1
+					)}`;
+				},
+				details: () => {
+					return document.location.href.split("/")[4].split("-").join(" ");
+				},
+				buttons: () => [
+					{
+						label: "View Course",
+						url: (() => {
+							return document.location.href.split("/").slice(0, 5).join("/");
+						})(),
+					},
+				],
+			},
+			{
+				path: /^\/learn\/.+\/quiz\/*/,
+				smallImageKey: () => Assets.Writing,
+				smallImageText: () => "Learning",
+				state: () => {
+					return "Taking a Quiz";
+				},
+				details: () => {
+					return document.location.href.split("/")[4].split("-").join(" ");
+				},
+				buttons: () => [
+					{
+						label: "View Course",
+						url: (() => {
+							return document.location.href.split("/").slice(0, 5).join("/");
+						})(),
+					},
+				],
+			},
+			{
+				path: /^\/learn\/.+\/exam\/*/,
+				smallImageKey: () => Assets.Writing,
+				smallImageText: () => "Learning",
+				state: () => {
+					return "Solving an Exam";
+				},
+				details: () => {
+					return document.location.href.split("/")[4].split("-").join(" ");
+				},
+				buttons: () => [
+					{
+						label: "View Course",
+						url: (() => {
+							return document.location.href.split("/").slice(0, 5).join("/");
+						})(),
+					},
+				],
+			},
+			{
+				path: /^\/learn\/.+\/ungraded*\/*/,
+				smallImageKey: () => Assets.Writing,
+				smallImageText: () => "Learning",
+				state: () => {
+					return `Subject: ${document.title.substring(
+						0,
+						document.title.lastIndexOf(" ") - 1
+					)}`;
+				},
+				details: () => {
+					return document.location.href.split("/")[4].split("-").join(" ");
+				},
+				buttons: () => [
+					{
+						label: "View Course",
+						url: (() => {
+							return document.location.href.split("/").slice(0, 5).join("/");
+						})(),
+					},
+				],
+			},
+
+			{
+				path: /^\/learn\/.*\/?$/,
+				smallImageKey: () => Assets.Viewing,
+				smallImageText: () => "Viewing",
+				details: () => `
+				Viewing a Course`,
+				state: () => {
+					return document.querySelector("[data-e2e='hero-title']").textContent;
+				},
+				buttons: () => [{ label: "Course Link", url: location.href }],
```

Comment:
Use `document.location` rather than `window.location` or `location` for consistency.
```suggestion
				buttons: () => [{ label: "Course Link", url: document.location.href }],
```